src/Command/Processor/KeyPrefixProcessor.php

```diff
@@ -181,9 +181,11 @@ public function __construct($prefix)
             /* ---------------- Redis 5.0 ---------------- */
             'XADD' => $prefixFirst,
             'XRANGE' => $prefixFirst,
+            'XREVRANGE' => $prefixFirst,
             'XDEL' => $prefixFirst,
             'XLEN' => $prefixFirst,
             'XACK' => $prefixFirst,
+            'XTRIM' => $prefixFirst,
```

Comment:
Would you mind adding tests for these? `KeyPrefixProcessorTest`

Reply:
Sure, done :slightly_smiling_face: